frontend/src/component/common/StrategyItemContainer/StrategyItemContainer.tsx

```diff
@@ -49,102 +57,101 @@ const StyledTruncator = styled(Truncator)(({ theme }) => ({
+}));
+
+const StyledHeaderInner = styled('div')(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    gap: theme.spacing(1),
+}));
 
 export const StrategyItemContainer: FC<StrategyItemContainerProps> = ({
     strategy,
     onDragStart,
     onDragEnd,
     headerItemsRight,
+    headerItemsLeft,
     strategyHeaderLevel = 3,
     children,
     style = {},
+    className,
 }) => {
     const StrategyHeaderLink: React.FC<{ children?: React.ReactNode }> =
-        'links' in strategy // todo: revisit this when we get to playground, related to flag `flagOverviewRedesign`
+        'links' in strategy
             ? ({ children }) => <Link to={strategy.links.edit}>{children}</Link>
             : ({ children }) => <> {children} </>;
 
     return (
         <Box sx={{ position: 'relative' }}>
-            <StyledContainer style={style}>
-                <StyledHeader
-                    draggable={Boolean(onDragStart)}
-                    disabled={Boolean(strategy?.disabled)}
-                >
-                    <ConditionallyRender
-                        condition={Boolean(onDragStart)}
-                        show={() => (
-                            <DragIcon
-                                draggable
-                                disableRipple
-                                size='small'
-                                onDragStart={onDragStart}
-                                onDragEnd={onDragEnd}
-                                sx={{ cursor: 'move' }}
-                            >
-                                <DragIndicator
-                                    titleAccess='Drag to reorder'
-                                    cursor='grab'
-                                    sx={{ color: 'action.active' }}
-                                />
-                            </DragIcon>
-                        )}
-                    />
-                    <StrategyHeaderLink>
-                        <StyledHeaderContainer>
-                            {strategy.title ? (
-                                <>
-                                    <p className='strategy-name'>
+            <StyledContainer style={style} className={className}>
+                <StyledHeader disabled={Boolean(strategy?.disabled)}>
+                    {onDragStart ? (
+                        <DragIcon
+                            draggable
+                            disableRipple
+                            size='small'
+                            onDragStart={onDragStart}
+                            onDragEnd={onDragEnd}
+                            sx={{ cursor: 'move' }}
+                        >
+                            <DragIndicator
+                                titleAccess='Drag to reorder'
+                                cursor='grab'
+                                sx={{ color: 'action.active' }}
+                            />
+                        </DragIcon>
```

Comment:
I think attempting to extract "draggable" feature from this into another container makes sense. It's a separate functionality and has a lot of complexity. Right now behind a flag it is broken for me on `main` and on this branch

Reply:
Yeah, this is a good point. It might. I did notice dragging was acting weird, but wasn't sure what was up. Considering this is already wonky on main, I'd rather merge this first and try to solve the dragging problem in a followup, though. Makes it easier to isolate the solution and get that merged.